kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTab.js

```diff
@@ -104,6 +60,12 @@ define(['bluebird', 'kb_common/html', 'common/ui', 'util/jobLogViewer', './jobSt
     }
 
     return {
+        /**
+         * Requires a Props object with the current jobState object at
+         * `exec.jobState`
```

Comment:
This module is called by the bulk import cell, and the `config` object gets passed in from that. You should be able to access a `jobState` object by running

```js
config.model.getItem('exec.jobState')
```

(the jobState is the only thing that this widget uses -- although once we have things hooked up to the backend, it may be that the job params are also available in it)